lessons/Early-Math/distributive-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,150 @@
+
+
+In the next activity, students will use Circles to explore how mathematical structure shifts when we apply Distributive Property.
+
+[.lesson-instruction]
+- On @printable-exercise{sum-to-product.adoc}, fill in the blanks so that the Circle on the right is equivalent to the Circle on the left.
+- Use computation to confirm that the Circles in each pair are equivalent.
+- After you finish evaluating each Circle, place a checkmark next to the Circle that you think results in a simpler comuptation.
+- When was it easier to compute the _sum_? When was it easier to compute the _product_?
+** _There are no right or wrong answers, here. We want students to begin to consider how a nuanced understanding of the Distributive Property can result in more flexible and efficient mental math._
+- On @printable-exercise{product-to-sum.adoc}, the Circle representing a _product_ is provided, and you will complete the Circle that represents a sum.
+
+=== Investigate
+
+Now Elise and Dani are discussing another pair of Circles.
+
+[.embedded, cols="^.^3,^.^3", grid="none", stripes="none" frame="none"]
+|===
+| @show{(coe '(* 5 43))} | @show{(coe '(+ 200 8))}
+|===
+
+Dani thinks the Circles are equivalent. She says, "You can think of 43 as the sum of 40 and 3. Then, you multiply 5 by 40 to get 200, and you add 5 and 3 to get 8."
+
+Elise says, "The second Circle should say 200 + 15, not 200 + 8."
+
+[.lesson-instruction]
+- Who is correct - Elise or Dani?
+** _Elise is correct. We need to multiply each value by 5._
+** _Note - it may be helpful for some students to see the Circle for @math{(5 \times 40) + (5 \times 3)}._
+
+The activities in the Launch focused on students' internalizing the structure of expressions both before and after applying the Distributive Property. During this segment of the lesson, students apply what they have learned.
+
+[.lesson-instruction]
+- Turn to @printable-exercise{true-or-false.adoc}, where you will determine if equations represented by Circles are true or false. Describe what you would change to make the false equations true.
+- Next, try @printable-exercise{wodb.adoc}. Cross out the Circle that doesn’t belong with the others, and then explain your choice.
+
+
+=== Synthesize
+
+- A common idea about arithmetic is that we must _always_ evaluate what's inside the parentheses first. Do you agree with this idea? Explain.
+- Summarize the Distributive Property in your own words.
+
+== The Distributive Property and Mental Math @duration{20 minutes}
+
+=== Overview
+
+=== Launch
+
+[.lesson-instruction]
+--
+- Look at this expression: @math{45 \times 81 - 45 \times 79}
+- Can we simplify this to an equivalent expression? How?
+** _We can apply the Distributive Property, turning this into @math{45 \times (81 - 79)}._
+- Can you represent your solving process with Circles?
+** _Invite a student to draw a Circle representation on the board. See below for a possible Circle representation._
```

Comment:
Not sure how to reword this but we don't need to introduce the new term Circle representation on top of Circles of Evaluation.